### DISCUSSION TITLE
How can we integrate AMP ( Accelerated Mobile pages) In nuxt 3.8 ?

### DISCUSSION THREAD

**Discussion by Ganeshreddykatla** [nuxt/nuxt#25387] [General]
Hi Team,

As we transition from Nuxt 2 to Nuxt 3.8, we previously utilized @nuxt/amp for generating AMP pages in the old project. Could you please provide guidance, documentation, or an example project on how to seamlessly integrate AMP into the new version?

Thank you.

**Comment by arashsheyda** [comment]
@Ganeshreddykatla what about using [Vite PWA](https://vite-pwa-org.netlify.app/frameworks/nuxt.html)?
